Meta.svelte

```diff
@@ -3,15 +3,74 @@
   Metadata helper
 -->
 <script>
-  /** Page title */
+  /** Page title **/
   export let title: Optional<string> = undefined;
-  /** Page description */
+  /** Page description **/
   export let description: Optional<string> = undefined;
-  /** Page image */
+  /** Page image **/
   export let image: Optional<string> = undefined;
+  /** Page url **/
+  export let url: Optional<string> = undefined;
+  /** Site logo **/
+  export let logoUrl: string = url ? `${url}/logo.png` : undefined;
+  /** Search URL **/
+  export let searchUrl: Optional<string> = undefined;
+  /** Sitemap URL **/
+  export let sitemapUrl: string = url ? `${url}/sitemap.xml` : undefined;
+  /** Twitter Options **/
+  export let twitter: any = {
```

Comment:
Wdyt about instead just giving freeform `key:value` options, since these are all just very light sugar on individual props (unlike title, description, etc)

eg: 

```
<Meta custom={{
 'twitter:card': 'summary_large_image',
 'og:siteName': 'en_US'
}} />
```

Reply:
I thought we could write interfaces for OpenGraph and Twitter to prevent typos.

Reply:
Hmm good call. Maybe we can leave Twitter and OG interfaces for another PR, I think it needs a bit of thinking.

Maybe we can still make it more abstracted with some sugar, etc

```
{
 twitter: {
   card: '',
   image: ''
  },
  openGraph: {
    ...
  }
}
```

Reply:
If the sub props include all possible og tags/are arbitrary, people could also use it to override the default page title for a particular network for eg